### PR TITLE
Use REST API instead of XML-RPC

### PR DESCRIPTION
**Pull request by hluk**

### CODE REVIEW

File: client.py [hunk 1]
```diff
@@ -1,26 +1,47 @@
...
 
 SERVERS = {
     'brew': 'https://brewhub.engineering.redhat.com/brewhub',
-    'localhost': 'http://localhost:5000/xmlrpc',
+    'localhost': 'http://localhost:5000/api/v1.0',
 }
 
 parser = argparse.ArgumentParser()
 parser.add_argument('server', choices=SERVERS.keys())
+parser.add_argument('--product', default='RHEL-6-Server-EXTRAS-6')
```
Comment (ktdreyer) [inline client.py]: really great argparse changes here. Mind breaking this into a separate commit?

File: product_listings_manager/__init__.py [hunk 2]
```diff
@@ -1,35 +1,36 @@
...
-        app.logger.error(err)
-        raise Fault(e)
+blueprint = Blueprint('api_v1', __name__)
+api = Api(blueprint)
+app.register_blueprint(blueprint, url_prefix='/api/v1.0')
+
+
+class About(Resource):
+    def get(self):
+        return {'version': __version__}
```
Comment (ktdreyer) [inline product_listings_manager/__init__.py]: AWESOME

File: product_listings_manager/__init__.py [hunk 3]
```diff
@@ -1,35 +1,36 @@
-import traceback
-from flask import Flask
-from flaskext.xmlrpc import XMLRPCHandler, Fault
+__version__ = '0.0.1'
```
Comment (ktdreyer) [inline product_listings_manager/__init__.py]: I've tagged v0.1.0, so let's make `__version__` align with that: `'0.1.0'`

Comment (hluk) [reply product_listings_manager/__init__.py]: Fixed.

File: client.py (hunk 1 above)
Comment (hluk) [reply client.py]: I removed it from this PR and will create new PR later just for changes in `client.py`.

**Comment by hluk** [comment]
I've split XML-RPC and REST API to separate submodules and added tests for REST API.

**Comment by ktdreyer** [comment]
This is really great. Thank you.

**Pull request merged**